contents.tex

```diff
@@ -0,0 +1,73 @@
+\mode*
+
+\section{Introduction}%
+\label{Introduction}
+
+In this lab we will evaluate some tools for private communication, in 
+particular, email and instant messaging.
+
+Email security has the longest history, this has been around since the 80s.
+Have they gained wide-spread adoption yet?
+No.
+There are two competing approaches: Pretty Good Privacy (PGP) and S/MIME 
+(comparable to HTTPS for websites).
```

Comment:
Why need to explain S/MIME but not PGP?

Reply:
Good point. I wrote it because you can also use GPG in S/MIME format. But that's not what I meant. By S/MIME I mean using X.509 certificates (same as for HTTPS), which seems to be the standard interpretation ([cf. Wikipedia](https://en.wikipedia.org/wiki/S/MIME)). I just wanted to emphasize this. Maybe I just remove the parentheses.